proposals/distribution.md

```diff
@@ -0,0 +1,156 @@
+
+    * What: Specifying a distribution method
+    * In/Out/Future: In scope
+    * Status: In progress (see opencontainers/distribution-spec)
+    * Description: Define a protocol for creating, retrieving, updating, and deleting objects defined in the [image specification][image-spec].
+        Listing repositories (like [`/v2/_catalog`][catalog]) is a multi-[image-index][] action, which is out of scope for this entry.
+        Managing groups of image indexes requires multi-[image-index][] actions, which are out of scope for this entry.
+        Listing image indexes within a group is a multi-[image-index][] action, which is out of scope for this entry.
+    * Why: This specification will provide one (of possibly many) distribution specifications.
+        Alternative distribution specifications may be developed for uses cases not covered by this specification, but defining them is currently out of scope for the OCI.
+
+* “Retrieving image content by its content-addressable hash”.
+    Docker's registry API already provides [endpoints for fetching manifest objects by digest][get-manifest].
+    Docker's registry API does not currently provide endpoints for fetching [image-index][] objects by digest, but this is the project where that will happen.
+
+    * What: Retrieving image content by its content-addressable hash
+    * In/Out/Future: In scope
+    * Status: In progress (see opencontainers/distribution-spec)
+    * Description: Specify a protocol for retrieving an [image index][image-index], [manifest][], or other [image specification][image-spec] object from a distribution engine by its content-addressable hash.
+    * Why: Using a hash as a name is a way to ensure a unique image name without relying on a particular naming authority or system.
+        Using hashing for name is an acceptable addition as it does not encode any centralized namespace.
+
+The following entries should remain in the [scope table][scope] but not be addressed by this project:
+
+* “Specifying way to attach signatures”.
+    We don't need to address this as part of distribution, because all resources are content-addressable and can be signed in external systems.
+
+## Related GitHub Issues
+
+* Simplifies tag listing: docker/distribution#2169
+* Allows listing of manifests: docker/distribution#2199
+
+[api.md]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md
+[catalog]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md#catalog
+[get-manifest]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md#pulling-an-image-manifest
+[iana-auth]: http://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml
+[image-spec]: https://github.com/opencontainers/image-spec/
+[image-index]: https://github.com/opencontainers/image-spec/blame/v1.0.1/image-index.md
```

Comment:
Any reason for the blame links?

Reply:
heyo! I think this was explained a bit up here --> https://github.com/opencontainers/tob/pull/35#discussion_r171951003 (I had the same question!)

Reply:
By that explanation this link does not need to be a blame. The other blame link to the same doc has an anchor, but it might be wise to consider not using blame and rather pointing to the parent section so the link is navigable to the sub sections.

Reply:
> By that explanation this link does not need to be a blame.

Good point.  I've filed #47 against this PR to fix this link.

> The other blame link to the same doc has an anchor, but it might be wise to consider not using blame and rather pointing to the parent section so the link is navigable to the sub sections.

I prefer linking [directly to `manifests`][1] instead of linking to [the whole section][2] (~85 lines of Markdown), but if @caniszczyk and/or the TOB prefer section links I'll survive.

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.1/image-index.md#L23
[2]: https://github.com/opencontainers/image-spec/blob/v1.0.1/image-index.md#image-index-property-descriptions